src/Web/Masa.Auth.Web.Admin.Server/Program.cs

```diff
@@ -24,7 +25,17 @@
 builder.Services.AddMasaStackComponentsForServer("wwwroot/i18n", publicConfiguration.GetValue<string>("$public.AppSettings:AuthClient:LocalUrl"), publicConfiguration.GetValue<string>("$public.AppSettings:McClient:Url"));
 builder.Services.AddSingleton<AddStaffValidator>();
 builder.Services.AddTypeAdapter();
-builder.Services.AddMasaOpenIdConnect(publicConfiguration.GetSection("$public.OIDC:AuthClient").Get<MasaOpenIdConnectOptions>());
+
+var masaOpenIdConnectOptions = publicConfiguration.GetSection("$public.OIDC:AuthClient").Get<MasaOpenIdConnectOptions>();
+builder.Services.AddMasaOpenIdConnect(masaOpenIdConnectOptions);
+builder.Services.AddJetTokenValidator(options =>
+{
+    options.AuthorityEndpoint = masaOpenIdConnectOptions.Authority;
```

Comment:
I think, if masaOpenIdConnectOptions?.Authority is not null, it can be autowired by AddMasaOpenIdConnect.